Products/MimetypesRegistry/mime_types/mtr_mimetypes.py

```diff
@@ -80,7 +80,8 @@ class text_xml(MimeTypeItem):
     binary = 0
 
     def classify(self, data):
-        m = re.search('^\s*<\\?xml.*\\?>', data)
+        pat = b'^\\s*<\\?xml.*\\?>' if isinstance(data, bytes) else '^\\s*<\\?xml.*\\?>'
+        m = re.search(pat, data)
```

Comment:
`return re.search(pat, data)`?

Reply:
re.search returns None or a pattern match object, but the original author returned 1 instead of the pattern match. I have no idea whether your suggested change has some overall impact, but i'd prefer to postpone logical changes until base py3 support is done.

Reply:
at least i turned the existing code into a one-liner now ;)